userspace/libsinsp/cri.cpp

```diff
@@ -143,6 +143,33 @@ bool cri_interface::parse_cri_image(const runtime::v1alpha2::ContainerStatus &st
 		}
 	}
 
+	if(image_name.empty() || strncmp(image_name.c_str(), "sha256", 6) == 0)
+	{
+		/* Retrieve image_name from annotations as backup when image name may start with sha256,
+		brute force try each schema we know of for containerd and crio container runtimes. */
+
+		Json::Value root;
+		Json::Reader reader;
+		if(reader.parse(info.find("info")->second, root))
```

Comment:
One question: what happens if "info" is not found in `info` (uhh... naming? :D)? Just for testing, can you change "info" to "whatever" to simulate an unsupported runtime and see if it crashes?

Reply:
Added, the other case you pointed out are fine. Also missed that when I added `container.cni.json`, therefore aded checks for both cases. Thanks @gnosek!